testsuite/tests/shapes/comp_units.ml

```diff
@@ -183,166 +182,6 @@ end
- "equal"[value] -> CU Stdlib . "Unit"[module] . "equal"[value];
- "t"[type] -> CU Stdlib . "Unit"[module] . "t"[type];
- "to_string"[value] -> CU Stdlib . "Unit"[module] . "to_string"[value];
- }
-type t = unit = ()
-val equal : t -> t -> bool = <fun>
-val compare : t -> t -> int = <fun>
-val to_string : t -> string = <fun>
-|}]
-
-module Without_constraint = Set.Make(Int)
-[%%expect{|
-{
- "Without_constraint"[module] ->
-   CU Stdlib . "Set"[module] . "Make"[module](CU Stdlib . "Int"[module])<.9>;
- }
-module Without_constraint :
-  sig
-    type elt = Int.t
-    type t = Set.Make(Int).t
-    val empty : t
-    val add : elt -> t -> t
-    val singleton : elt -> t
-    val remove : elt -> t -> t
-    val union : t -> t -> t
-    val inter : t -> t -> t
-    val disjoint : t -> t -> bool
-    val diff : t -> t -> t
-    val cardinal : t -> int
-    val elements : t -> elt list
-    val min_elt : t -> elt
-    val min_elt_opt : t -> elt option
-    val max_elt : t -> elt
-    val max_elt_opt : t -> elt option
-    val choose : t -> elt
-    val choose_opt : t -> elt option
-    val find : elt -> t -> elt
-    val find_opt : elt -> t -> elt option
-    val find_first : (elt -> bool) -> t -> elt
-    val find_first_opt : (elt -> bool) -> t -> elt option
-    val find_last : (elt -> bool) -> t -> elt
-    val find_last_opt : (elt -> bool) -> t -> elt option
-    val iter : (elt -> unit) -> t -> unit
-    val fold : (elt -> 'acc -> 'acc) -> t -> 'acc -> 'acc
-    val map : (elt -> elt) -> t -> t
-    val filter : (elt -> bool) -> t -> t
-    val filter_map : (elt -> elt option) -> t -> t
-    val partition : (elt -> bool) -> t -> t * t
-    val split : elt -> t -> t * bool * t
-    val is_empty : t -> bool
-    val mem : elt -> t -> bool
-    val equal : t -> t -> bool
-    val compare : t -> t -> int
-    val subset : t -> t -> bool
-    val for_all : (elt -> bool) -> t -> bool
-    val exists : (elt -> bool) -> t -> bool
-    val to_list : t -> elt list
-    val of_list : elt list -> t
-    val to_seq_from : elt -> t -> elt Seq.t
-    val to_seq : t -> elt Seq.t
-    val to_rev_seq : t -> elt Seq.t
-    val add_seq : elt Seq.t -> t -> t
-    val of_seq : elt Seq.t -> t
-  end
-|}]
-
-module With_identity_constraint : sig
-  module M : Set.S
-end = struct
-  module M = Set.Make(Int)
-end
-[%%expect{|
-{
- "With_identity_constraint"[module] ->
-   {<.12>
-    "M"[module] ->
-      CU Stdlib . "Set"[module] . "Make"[module](
-      CU Stdlib . "Int"[module])<.10>;
-    };
- }
-module With_identity_constraint : sig module M : Set.S end
-|}]
-
-module With_constraining_constraint : sig
-  module M : sig type t end
-end = struct
-  module M = Set.Make(Int)
-end
-[%%expect{|
-{
- "With_constraining_constraint"[module] ->
-   {<.16>
-    "M"[module] ->
-      {<.13>
-       "t"[type] ->
-         CU Stdlib . "Set"[module] . "Make"[module](
-         CU Stdlib . "Int"[module])<.13> . "t"[type];
-       };
-    };
->>>>>>> ocaml/5.1
```

Comment:
Any reason we shouldn't just take `tip-5` here? (Do you know a reason why we wouldn't want to test this?)